reports/graphs/feeding_amounts.py

```diff
@@ -14,23 +14,57 @@ def feeding_amounts(instances):
     :param instances: a QuerySet of Feeding instances.
     :returns: a tuple of the the graph's html and javascript.
     """
-    totals = {}
+    feeding_types = [
+            'breast milk',
+            'formula',
+            'fortified breast milk',
+            'solid food',
+        ]
+    feeding_types_desc = [
```

Comment:
See previous comment re: model field metadata. These descriptive names will come from the same place.